data/json/vehicle_parts.json

```diff
@@ -3679,7 +3679,7 @@
         "range" : 12,
         "fuel_type" : "40mm",
         "item" : "mark19",
-        "difficulty" : 6,
+        "difficulty" : 4,
```

Comment:
Burst-fire 40mm launcher attached to uh, ...wherever the PC wants, ought to take some doing, I'd think.  5 seems better.